modules/checklist/assets/js/components/app.js

```diff
@@ -0,0 +1,24 @@
+import ReactUtils from 'elementor-utils/react';
+import Checklist from './checklist';
+
+export const App = () => {
+	let container = document.getElementById( 'e-checklist' );
+
+	if ( ! container ) {
+		container = document.createElement( 'div' );
+		container.id = 'e-checklist';
+		document.body.append( container );
+	}
+
+	const isRTL = elementorCommon.config.isRTL;
+
+	if ( ! container.hasChildNodes() ) {
+		ReactUtils.render( (
+			<Checklist
+				isRTL={ isRTL }
+			/>
+		), container );
+	}
+
+	elementor.trigger( 'elementor/editor/panel/checklist/clicked' );
+};
```

Comment:
Thee app file doesn't belong tin the component folder, it should be in the same level as the folder

Reply:
I see, I wonder if that's on purpose, in any way, as far as I know, the app is usually above the components but I don't mind leaving it as is. Let's ask tomorrow at the daily if that matters at all

Reply:
I also prefer `app.js` to be outside of the `components` folder 🙏